docs/user/data_management.rst

```diff
@@ -186,6 +186,30 @@ user data using the hint ``geomesa.z3.interval``:
 
 See below for alternate ways to set the user data.
 
+.. _customizing_xz_index:
+
+Customizing the XZ-Index
```

Comment:
Do we explain elsewhere when a 'XZ' index is used?  Is it worth explaining that any XZ2 / XZ3 tables created by will use this config?